src/stdlib/map.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Map
+---
+
+A Map holds key-value pairs. Any value may be used as a key or value.
+
+```grain
+import Map from 'map'
+```
+
+## Values
+
+### Map.**make**
+
+```grain
+make : () -> Map<k, v>
+```
+
+Creates a new, empty map.
+
+### Map.**makeSized**
+
+```grain
+makeSized : Number -> Map<k, v>
+```
+
+Creates a new, empty map with an initial storage size for the given number of elements.
+
+### Map.**get**
+
+```grain
+get : (k, Map<k, v>) -> v
+```
+
+Retrieves the value associated with the given key from the map.
+
+### Map.**set**
+
+```grain
+set : (k, v, Map<k, v>) -> Void
+```
+
+Adds a new key-value pair to the map.
```

Comment:
I forget, does it replace the value if it is already there?

Reply:
Yeah, it replaces.

Reply:
```suggestion
Adds a new key-value pair to the map. If the key already exists, replaces the value.
```